driver/tests.py

```diff
@@ -42,32 +43,37 @@ def test_commandline_args():
 
 def test_aliases():
     for alias, config in ALIASES.items():
-        cmd = ["./fast-downward.py", "--alias", alias, "output.sas"]
+        cmd = [sys.executable, "fast-downward.py", "--alias", alias, "output.sas"]
         run_driver(cmd)
 
 
 def test_show_aliases():
-    run_driver(["./fast-downward.py", "--show-aliases"])
+    run_driver([sys.executable, "fast-downward.py", "--show-aliases"])
 
 
 def test_portfolios():
     for name, portfolio in PORTFOLIOS.items():
-        cmd = ["./fast-downward.py", "--portfolio", portfolio,
+        cmd = [sys.executable, "fast-downward.py", "--portfolio", portfolio,
                "--search-time-limit", "30m", "output.sas"]
         run_driver(cmd)
 
 
 def test_hard_time_limit():
+    # We cannot test this on systems like Windows where we cannot
+    # enforce time limits.
+    if not limits.can_set_time_limit():
+        return
```

Comment:
We could use the decorator `@pytest.skipif(not limits.can_set_time_limit())` for this, I think. That gives us nicer test output and is "more standard".